src/site/content/en/blog/usb/index.md

```diff
@@ -283,6 +283,47 @@ provides a ground-up example of building a USB-controlled LED device designed
 for the WebUSB API (not using an Arduino here). You'll find hardware, software,
 and firmware.
 
+### Limits on transfers size
+
+Some operating systems impose limitations on how much data can be part of
+pending USB transactions. Splitting your data into smaller transactions and only
+submitting a few at a time helps avoiding those limitations. It also reduces
+the amount of memory used and allows your application to report progress as the
+transfers complete.
+
+As multiple transfers submitted to an endpoint always execute in order, it is
```

Comment:
```suggestion
Because multiple transfers submitted to an endpoint always execute in order, it is
```

Reply:
Most of the time, using 'as' as a synonym of 'because' is not a problem. In this case, I read it as a synonym of 'while' and had to start the sentence again when I got the main clause.

Reply:
Noted. Thank you!